plugins/in_calyptia_fleet/in_calyptia_fleet.c

```diff
@@ -258,7 +210,12 @@ static flb_sds_t fleet_config_filename(struct flb_in_calyptia_fleet_config *ctx,
         return NULL;
     }
 
-    ret = flb_sds_printf(&cfgname, PATH_SEPARATOR "%s.conf", fname);
+    if (ctx->fleet_config_legacy_format) {
+        ret = flb_sds_printf(&cfgname, PATH_SEPARATOR "%s.conf", fname);
+    } else {
```

Comment:
`else` should start in a new line: https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#braces-usage-on-conditionals-loops-and-functions